rust-lib/src/lib.rs

```diff
@@ -346,3 +355,51 @@ pub extern "C" fn transfer(
     )
     .into()
 }
+
+fn _asset_allocations(
+    runtime: &COpaqueStruct,
+    contract_id: *mut c_char,
+) -> Result<(), RequestError> {
+    let runtime = Runtime::from_opaque(runtime)?;
+
+    let c_contract_id = unsafe { CStr::from_ptr(contract_id) };
+    let contract_id = ContractId::from_str(c_contract_id.to_str()?)?;
+
+    info!("AssetAllocationsArgs {{ contract_id: {:?} }}", contract_id);
```

Comment:
Don't know why Alekos used `info!` in these functions here, this is clearly a `debug!` type of information

Reply:
Seems like this is still have to be `debug!`, as was discussed regarding the criteria. This is read-only method, so no need to provide verbose output about it's args

Reply:
as already agreed, I will change this when facing #23